### PR TITLE
Lit Support | Pulac-Cerullo Reminders

### PR DESCRIPTION
**Pull request by jcq**
Connects #11256

### Description
This updates the copy for the existing Pulac-Cerullo modal, and adds additional reminder alerts to prompt users to check CAVC for jurisdictional conflicts 

### Acceptance Criteria
- [ ] Code compiles correctly

### CODE REVIEW

File: client/app/pulacCerullo/CavcLinkInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (leikkisa) [inline client/app/pulacCerullo/CavcLinkInfo.jsx]: I think this whole folder should go into the queue components folder

File: client/app/pulacCerullo/PulacCerulloReminderModal.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    value: 'yes'
+  }
+];
+
+export const PulacCerulloReminderModal = ({ onSubmit, onCancel }) => {
+  const [hasCavc, setHasCavc] = useState(null);
+
+  const cancelHandler = () => onCancel();
+  const submitHandler = () => {
+    //   Should we perform / return separate actions? Likely just return the selection
```
Comment (leikkisa) [inline client/app/pulacCerullo/PulacCerulloReminderModal.jsx]: I think it makes sense to return the selection

File: spec/feature/queue/task_queue_spec.rb [hunk 3]
```diff
@@ -301,38 +301,58 @@
...
-      find("div", class: "Select-option", text: label).click
+      validate_pulac_cerullo_tasks_created_success_msg(label)
 
-      fill_in("taskInstructions", with: instructions)
-      find("button", text: "Submit").click
+      click_dropdown(text: Constants.TASK_ACTIONS.LIT_SUPPORT_PULAC_CERULLO.label)
+      click_button(text: "Notify")
 
+      mail_task = root_task.reload.children[0]
+      validate_pulac_cerullo_tasks_created_mail_task(mail_task, task_class)
```
Comment (leikkisa) [inline spec/feature/queue/task_queue_spec.rb]: These method names are pretty confusing. The MTV mail task and it's child task were created before and are unrelated to the pulac-cerrulo

validate_pulac_cerullo_tasks_created_setup
validate_pulac_cerullo_tasks_created_mail_task

**Comment by leikkisa** [comment]
Screen Shot 2019-10-14 at 11 10 25 AM

Adding to my review, the above could use some styling

**Comment by leikkisa** [comment]
Also noticed this when working on it, I can create duplicate Pulac-cerrulo tasks:
Screen Shot 2019-10-14 at 11 18 06 AM

File: client/app/pulacCerullo/CavcLinkInfo.jsx (hunk 1 above)
Comment (jcq) [reply client/app/pulacCerullo/CavcLinkInfo.jsx]: That's odd — I thought that's where I had put it. Oops! Good catch.

File: spec/feature/queue/task_queue_spec.rb (hunk 3 above)
Comment (jcq) [reply spec/feature/queue/task_queue_spec.rb]: Honestly, I *really* didn't want to make those. This was all broken up in order to pass code complexity filters. :\

I could put them all back in and we could just bypass that check I suppose...?

**Comment by jcq** [comment]
> Also noticed this when working on it, I can create duplicate Pulac-cerrulo tasks:
> Screen Shot 2019-10-14 at 11 18 06 AM

What were the steps that allowed you to do this?

**Pull request merged**